### PR TITLE
Add `Craft` project pages

### PR DESCRIPTION
**Pull request by joshlory**
+cc @ryansloan to verify we have sign-off on /p/mc and /p/minecraft URLs, or suggest new ones.

Base levels:
https://studio.code.org/s/mc/stage/1/puzzle/14
https://studio.code.org/s/minecraft/stage/1/puzzle/12

### CODE REVIEW

File: dashboard/app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -27,6 +27,12 @@ class ProjectsController < ApplicationController
     gumball: {
       name: 'New Gumball Project'
     },
+    mc: {
+      name: 'New Minecraft Simple Project'
```
Comment (Hamms) [inline dashboard/app/controllers/projects_controller.rb]: Isn't this now called "Minecraft Adventurer"?

File: dashboard/config/scripts/levels/New Minecraft Simple Project.level [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    "examples_required": "false",
+    "definition_highlight": "false",
+    "definition_collapse": "false",
+    "disable_examples": "false",
+    "free_play": "true",
+    "available_blocks": "[\"bedrock\", \"bricks\", \"oreCoal\", \"dirtCoarse\", \"cobblestone\", \"oreDiamond\", \"dirt\", \"oreEmerald\", \"farmlandWet\", \"glass\", \"oreGold\", \"grass\", \"gravel\", \"oreIron\", \"oreLapis\", \"logAcacia\", \"logBirch\", \"logJungle\", \"logOak\", \"logSpruce\", \"planksAcacia\", \"planksBirch\", \"planksJungle\", \"planksOak\", \"planksSpruce\", \"oreRedstone\", \"sand\", \"sandstone\", \"stone\", \"tnt\", \"wool\"]",
+    "if_block_options": "[\"water\", \"lava\", \"\"]",
+    "player_start_direction": "0",
+    "special_level_type": "freeplay",
+    "instructions": "Congratulations! You've completed all our puzzles! Now mine, build, and create something unique by using everything you've learned so far."
```
Comment (Hamms) [inline dashboard/config/scripts/levels/New Minecraft Simple Project.level]: is this still valid?

File: dashboard/app/controllers/projects_controller.rb (hunk 1 above)
Comment (joshlory) [reply dashboard/app/controllers/projects_controller.rb]: Good catch, I'll update the project name and URLs.  This isn't public-facing, but the URLs will be.

**Comment by joshlory** [comment]
PTAL.

**Pull request merged**